lib/rules/number-max-precision/README.md

```diff
@@ -37,3 +37,57 @@ a { top: 3.24px; }
 ```css
 @media (min-width: 3.23em) {}
 ```
+
+## Optional secondary options
+
+### `ignoreUnits: ["string"]`
+
+Ignore number-max-precision rule for the value with the unit.
```

Comment:
Can we use the following description, please:

"Ignore the precision of numbers for values with the specified units."

As it's more consistent with the descriptions [else where](https://github.com/stylelint/stylelint/search?l=Markdown&p=1&q=ignore&type=&utf8=%E2%9C%93).